AsyncAwait.Task2.CodeReviewChallenge/Models/Support/ManualAssistant.cs

```diff
@@ -19,16 +18,13 @@ public async Task<string> RequestAssistanceAsync(string requestInfo)
     {
         try
         {
-            var t = _supportService.RegisterSupportRequestAsync(requestInfo);
-            Console.WriteLine(t.Status); // this is for debugging purposes
-            Thread.Sleep(5000); // this is just to be sure that the request is registered
-            return await _supportService.GetSupportInfoAsync(requestInfo)
-                .ConfigureAwait(false);
```

Comment:
This is not optimal as  _supportService.GetSupportInfoAsync relies on the execution of the _supportService.RegisterSupportRequestAsync task and the to methods should be awaited consecutively. This allows us to remove the Thread.Sleep statement and gain performance benefit. Also writing to the console has also negative performance consequences, that`s why it is removed.